cfgagent/vlanconfd.cpp

```diff
@@ -0,0 +1,88 @@
+#include <unistd.h>
+#include <vector>
+#include "dbconnector.h"
+#include "select.h"
+#include "exec.h"
+#include "schema.h"
+#include "macaddress.h"
+#include "producerstatetable.h"
+#include "switchconf.h"
+#include "vlanconf.h"
+
+using namespace std;
+using namespace swss;
+
+/* select() function timeout retry time, in millisecond */
+#define SELECT_TIMEOUT 1000
+
+MacAddress gMacAddress;
+SwitchConf *gSwtichConfVlan;
+
+int main(int argc, char **argv)
+{
+    Logger::linkToDbNative("vlanconfd");
+    SWSS_LOG_ENTER();
+
+    SWSS_LOG_NOTICE("--- Starting vlanconfd ---");
+
+    string mac_str;
+    swss::exec("ip link show eth0 | grep ether | awk '{print $2}'", mac_str);
+    gMacAddress = mac_str;
```

Comment:
can we make this as an parameter instead of hard code the vlan mac address into the code logic, see -m option in orchagent?

Reply:
SWITCH_ATTR in SWITCH table has switch_mac field, to implement mac fetching from configDB SWITCH.  mac could be be taken from there if configured, otherwise eth0 mac is used by default.  
mac address here is to set on VLAN.